src/components/Posts/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+import { toast } from 'react-toastify';
+import { Hearts } from 'react-loader-spinner';
+
+import { getPosts } from 'api/post.service';
+
+import Post from 'components/Post';
+import Box from '../Box';
+
+import './style.scss';
+
+const Posts = () => {
+  const [isFetchingPosts, setIsFetchingPosts] = useState(true);
+  const [posts, setPosts] = useState([]);
+
+  useEffect(() => {
+    setIsFetchingPosts(true);
```

Comment:
This is duplicated. When you first enter the screen, the isFetchingPosts variable is already by default true